bin/ccd-def/create-ccd-caseworker-user.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+USER_EMAIL="${1:-me@server.net}"
```

Please give the user different default email address. Using the same one everywhere only makes things more confusing as operation silently fails due to email address collision.

---

changed appropriately